api-guidelines/rest/http/headers/rules/must-forward-test-header.md

```diff
@@ -0,0 +1,9 @@
+---
+id: R0000XX
+---
+
+# MUST forward `Test` header
+
+An API provider or consumer that makes API requests as a direct consequence of an API request or response flagged with a test header, must add the [`test` header](./may-use-header.md) of the triggering interaction to all following requests.
```

Comment:
Sorry for that, I know I introduced the comma in the first place. Shame on you past-Christina :D But we should consistently spell "Test" :)

```suggestion
An API provider or consumer that makes API requests as a direct consequence of an API request or response flagged with a test header must add the [`Test` header](./may-use-header.md) of the triggering interaction to all following requests.
```